### PR TITLE
test: add autoLaunch false for tvOS

### PR DESCRIPTION
**Pull request by KazuCocoa**
Let me add a basic test for tvOS

### CODE REVIEW

File: test/functional/tv/tvos-e2e-specs.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ describe('tvOS', function () {
     baseCaps = Object.assign({}, TVOS_CAPS, {udid});
     caps = Object.assign({usePrebuiltWDA: true}, baseCaps);
   });
-  after(async function () {
+  afterEach(async function () {
```
Comment (mykola-mokhnach) [inline test/functional/tv/tvos-e2e-specs.js:31]: Please review what should be done before and after each test case. Simulator shutdown and recreation is an expensive operation, which may significantly hit tests performance

Comment (KazuCocoa) [reply test/functional/tv/tvos-e2e-specs.js]: yeah

**Comment by KazuCocoa** [comment]
Run functional tests in e2e_tvos_AppleTV_13
before: 2m52s
after: 3m0s

File: test/functional/tv/tvos-e2e-specs.js [hunk 2]
```diff
@@ -18,31 +18,42 @@ describe('tvOS', function () {
...
-      { platform: 'tvOS' });
-    baseCaps = Object.assign({}, TVOS_CAPS, {udid});
-    caps = Object.assign({usePrebuiltWDA: true}, baseCaps);
+      { platform: TVOS_CAPS.platformName });
   });
+
   after(async function () {
-    const sim = await getSimulator(caps.udid, {
-      platform: 'tvOS',
+    const sim = await getSimulator(udid, {
```
Comment (mykola-mokhnach) [inline test/functional/tv/tvos-e2e-specs.js:31]: short-circuit here if udid is unset (setup failed)

**Pull request merged**